Right clicking the title bar area of the application crashes the application

**Describe the bug**

Right clicking the title bar area of the application crashes the application 

**Additional Context**

If you right click the upper part of the application, the so called title bar of the application. The application just crashes.

**Wabbajack Version**

4.0.0.3

**Logs**

```
00:00:04.645 [ERROR] (Wabbajack.MainWindow) Uncaught error|System.InvalidOperationException: Can only call DragMove when primary mouse button is down.
   at System.Windows.Window.DragMove()
   at Wabbajack.MainWindow.TitleBar_OnMouseDown(Object sender, MouseButtonEventArgs e) in e:\oss\wabbajack\Wabbajack.App.Wpf\Views\MainWindow.xaml.cs:line 196
   at Wabbajack.MainWindow.<.ctor>b__3_27(MouseButtonEventArgs x) in e:\oss\wabbajack\Wabbajack.App.Wpf\Views\MainWindow.xaml.cs:line 163
   at System.Reactive.AnonymousObserver`1.OnNextCore(T value)
   at System.Reactive.ObserverBase`1.OnNext(T value)
   at System.Reactive.AutoDetachObserver`1.OnNextCore(T value)
   at System.Reactive.ObserverBase`1.OnNext(T value)
   at System.Windows.Controls.RxGridEvents.<>c__DisplayClass75_0.<get_MouseDown>g__Handler|1(Object sender, MouseButtonEventArgs e) in e:\oss\wabbajack\Wabbajack.App.Wpf\obj\x64\Release\net9.0-windows\win-x64\ReactiveMarbles.ObservableEvents.SourceGenerator\ReactiveMarbles.ObservableEvents.SourceGenerator.EventGenerator40\SourceClassSystem.Windows.Controls.Grid-InstanceEvents.SourceGenerated.cs:line 229
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.UIElement.RaiseTrustedEvent(RoutedEventArgs args)
   at System.Windows.Input.InputManager.ProcessStagingArea()
   at System.Windows.Input.InputProviderSite.ReportInput(InputReport inputReport)
   at System.Windows.Interop.HwndMouseInputProvider.ReportInput(IntPtr hwnd, InputMode mode, Int32 timestamp, RawMouseActions actions, Int32 x, Int32 y, Int32 wheel)
   at System.Windows.Interop.HwndMouseInputProvider.FilterMessage(IntPtr hwnd, WindowMessage msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at System.Windows.Interop.HwndSource.InputFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at System.Windows.Threading.Dispatcher.Invoke(DispatcherPriority priority, Delegate method, Object arg)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
   at MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)
   at System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
   at System.Windows.Application.RunDispatcher(Object ignore)
   at System.Windows.Application.RunInternal(Window window)
   at Wabbajack.App.Main()
```

--- a/Wabbajack.App.Wpf/Views/MainWindow.xaml.cs
+++ b/Wabbajack.App.Wpf/Views/MainWindow.xaml.cs
@@ -193,7 +193,9 @@ public partial class MainWindow : MetroWindow
         if (Application.Current.MainWindow.WindowState == WindowState.Maximized)
             Application.Current.MainWindow.WindowState = WindowState.Normal;
 
-        DragMove();
+        if (e.LeftButton == MouseButtonState.Pressed) {
+            DragMove();
+        }
     }
 
     private void FloatingWindowBackground_OnMouseDown(object sender, MouseButtonEventArgs x)

